README.md

```diff
@@ -50,6 +52,13 @@ For examples and benchmarks, see our [blog](http://vertex.ai/blog).
 - [Reporting Issues](#reporting-issues)
 
 ### Recent Release Notes
+* PlaidML 0.3.3 - 0.3.5
+  * Support Keras 2.2.0
```

Comment:
As far as I can tell we support Keras 2.2.0-2.2.2 (and indeed I put a bit of effort into ensuring we are compatible with both, and the version we install (e.g. with pip) if the user doesn't do anything special is 2.2.2). I think it would be reasonable to claim compatibility with that whole range?